### ISSUE TITLE
🐛 `px-*` breaks tailwind parsing

### ISSUE THREAD

**Issue by sean-risr** [biomejs/biome#8708]
### Environment information

<details>

```bash

CLI:
  Version:                      2.3.11
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v25.2.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/11.6.2

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true
  HTML full support enabled:    unset

Workspace:
  Open Documents:               0


```

</details>

### What happened?

When creating a utility with a parameter (`-*`) in Tailwind v4, Biome's parser reports the error:
```
  ✖ expected `{` but instead found `-`

    160 │ 
  > 161 │ @utility px-* {
        │            ^
    162 │       padding-right: --value(--s-x-*);
    163 │       padding-left: --value(--s-x-*);
    164 | }
```
Escaping the characters with backslashes stops Biome from reporting the error, but breaks Tailwind.

Code:
```css
@theme {
	--w-topic-card: 20rem;
}

/* Define utilities to match custom values */
@utility w-* {
	width: --value(--w-*);
}
```

I've added the `tailwindDirectives` to my biome.json file without success.

### Expected result

Should not report an error

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
You say you're running 2.3.11, but I could've sworn I fixed this a few patches back. I also can't reproduce it in the playground https://biomejs.dev/playground/?code=QAB0AGgAZQBtAGUAIAB7AAoACQAtAC0AdwAtAHQAbwBwAGkAYwAtAGMAYQByAGQAOgAgADIAMAByAGUAbQA7AAoAfQAKAAoALwAqACAARABlAGYAaQBuAGUAIAB1AHQAaQBsAGkAdABpAGUAcwAgAHQAbwAgAG0AYQB0AGMAaAAgAGMAdQBzAHQAbwBtACAAdgBhAGwAdQBlAHMAIAAqAC8ACgBAAHUAdABpAGwAaQB0AHkAIAB3AC0AKgAgAHsACgAJAHcAaQBkAHQAaAA6ACAALQAtAHYAYQBsAHUAZQAoAC0ALQB3AC0AKgApADsACgB9AA%3D%3D&language=css

**Comment by sean-risr** [comment]
@dyc3 My bad, it was only the `px-*` that errors, not any of the other wildcard matchers.

Repro: https://biomejs.dev/playground/?code=QAB0AGgAZQBtAGUAIAB7AAoACQAtAC0AcwAtAHgALQBiAGEAcwBlAC0AYwBhAHIAZAA6ACAAMAAuADUAcgBlAG0AOwAKAAkALQAtAHMALQB5AC0AdABvAHAAaQBjAC0AYwBhAHIAZAA6ACAAMAAuADUAcgBlAG0AOwAKAH0ACgAKAEAAdQB0AGkAbABpAHQAeQAgAHAAeQAtACoAIAB7AAoACQBwAGEAZABkAGkAbgBnAC0AdABvAHAAOgAgAC0ALQB2AGEAbAB1AGUAKAAtAC0AcwAtAHkALQAqACkAOwAKAAkAcABhAGQAZABpAG4AZwAtAGIAbwB0AHQAbwBtADoAIAAtAC0AdgBhAGwAdQBlACgALQAtAHMALQB5AC0AKgApADsACgB9AAoACgBAAHUAdABpAGwAaQB0AHkAIABwAHgALQAqACAAewAKAAkAcABhAGQAZABpAG4AZwAtAHIAaQBnAGgAdAA6ACAALQAtAHYAYQBsAHUAZQAoAC0ALQBzAC0AeAAtACoAKQA7AAoACQBwAGEAZABkAGkAbgBnAC0AbABlAGYAdAA6ACAALQAtAHYAYQBsAHUAZQAoAC0ALQBzAC0AeAAtACoAKQA7AAoAfQA%3D&language=css

**Comment by sean-risr** [comment]
@dyc3 Can you please reopen this? It's confirmed and has a repro
